mne/io/bti/bti.py

```diff
@@ -1143,7 +1143,8 @@ def by_name(x):
         data = _read_data(bti_info)
         assert len(data) == len(info['ch_names'])
         self._projector_hashes = [None]
-        super(RawBTi, self).__init__(info, data, verbose=verbose)
+        super(RawBTi, self).__init__(
+            info, data, filenames=[pdf_fname], verbose=verbose)
```

Comment:
Not sure I get it. Would this work if you passed additional params like config and headshape for BTI?